src/Tool-Base/SystemNavigation.extension.st

```diff
@@ -538,10 +545,9 @@ SystemNavigation >> confirmRemovalOfSelectors: aCollection on: aClass [
 	collection ifEmpty: [ ^ 1 ].
 	count := collection sum: [ :sel | (self allCallsOn: sel) size ].
 	selectors := collection joinUsing: ', #' last: ' and #'.
-	"ugly but faster"
-	caption := count = 1
-		ifTrue: [ 'The messages #' , selectors , ' have ' , count printString , ' sender' ]
-		ifFalse: [ 'The messages #' , selectors , ' have ' , count printString , ' senders' ].
+	
+	caption := 'The <1?message:messages> #<2s> <1?has:have> <3p> <4?sender:senders' expandMacrosWithArguments: {collection size = 1. selectors. count. count = 1}.
```

Comment:
As I'm new to Smalltalk, could anyone explain use of "<?:>" in the string. Is this a feature of language or some kind of compiler extension?